test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -643,6 +659,43 @@ def _assert_artifact_free(output, stray_artifacts):
             artifact, output.stdout
         ), f"Matched {artifact} in {output.stdout} while running {output.command}"
 
+def _test_sm_toolkit_and_ts_version(image, region):
+    """
+    @param image: ECR image URI
+    """
+
+    if "gpu" or "cpu" in image:
+        cmd_smkit = "pip show sagemaker-pytorch-inference | grep -i Version"
+        cmd_ts = "torchserve --version"
+    else:
+        ValueError(
+            "Test as of now only covers CPU and GPU type images. If required, please modify this test to accommodate the new image type!")
```

Comment:
I suggest not hardcoding this to device type - instead I would drive decision to error based on whether packages are in the container. This test should only run on images that require the label

Reply:
fixed and no longer an issue